### ISSUE TITLE
Row system broken (?)

### ISSUE THREAD

**Issue by GeekinaCave** [cybrox/zythum#7]
hi, when using the the template for organize mods and reach 10 columns the order resets and appear at the beginning, how change that?

**Comment by cybrox** [comment]
I don't think you can change that. As far as I know, it is intended by Factorio that there are not more than 10 columns in one row.

**Comment by GeekinaCave** [comment]
Im sorry, i have a brain fart, i mean the row system: when i use the row 10 it resets an put it below row 1.
 the only way to make it work is extended the row system to 100 and configure the row i need in 90.
